CHANGELOG.md

```diff
@@ -1,3 +1,106 @@
+10.0.0 Release notes (yyyy-MM-dd)
+=============================================================
+
+This release is functionally identical to v10.0.0-rc.2.
+
+NOTE: This version upgrades the Realm file format to version 20. Realm files
```

Comment:
We should not be using the version of the file format (was agreed some time ago that this is not something we want exposed publicly). Instead, we should be using more generic terms like "The internal layout of the Realm file has changed. Realm files will be automatically upgraded when opened..."